alacritty.yml

```diff
@@ -6,7 +6,7 @@
 # loaded earlier with those loaded later in the chain. The file itself will
 # always be loaded last. If an import path cannot be found it will be skipped.
 #import:
-#  - /path/to/alacritty.yml
+#  - ~/path/to/alacritty.yml
```

Comment:
I'm not sure if it's clear that it's not shell syntax. Though maybe it's fine...

Reply:
I've tried to simplify the other section a little bit to document this properly. Hopefully it should be fine now.